app/controllers/account/admin/issues_controller.rb

```diff
@@ -37,13 +40,28 @@ def destroy
 
       private
 
+      def facebook_posting(issue)
+        return if Rails.env.test?
+        page = prepare_facebook_page
+        page.feed!(message: "Адреса: #{issue.location}. \n \n
```

Comment:
what if description is blank or no tags? you will post a lot of new-lines. refactor it a little